doc/PORTAL_STORM_BALANCE_AND_DESIGN.md

```diff
@@ -0,0 +1,68 @@
+# Portal Storm Design
+Portal storms are otherworldly colisions. Evil and malicious entities from inexplainable spaces are pouring into our world. However this doesn't mean it's a good way to imediately kill the character.
+
+## Ideas
+Portal Storms should consist of the inexplainable and the disturbing.  There should be less of a focus even on our usual nether monsters, instead leaning on even more disturbing creatures. 
+
+## Reminders
+Some things to keep in mind when working on Portal Storms:
+* Portal storms are weather: You are not writing boss fights, or end game encounters, you are writing and implementing weather effects. Try not to lose track of that.
+* Portal storms shouldn't be forced rest: It should be *possible* for a strong character to maneuver in a portal storm. It can be risky, but shouldn't be impossible.
+* Portal storm encounters don't need to be rewarding: Some encounters will be, but not every encounter needs a carrot. The variance of portal storms means that traveling the storms is a gamble, sometimes you get lucky, other times you do not.
+* Portal storms should only be *actively* dangerous with creatures hunting you when you go outside (more on this in balance).
+* Portal storms are trying to destroy humans, and warp the world: Portal storms should also really only be dangerous to humans, creatures that can percieve the otherworldly nature. Animals and the dead should have little to no interest in the other side and vice versa.
+* Don't be annoying: Nothing ruins ambience as fast as tedium. Don't spam effects, don't waste the players time. Portal Storms already *show* the player they are dangerous, we do not need to *tell* the player they are scary or weird.
```

Comment:
"Tedium" is pretty vague here and generally just used for heckling. Consider "repetition" instead.